### PR TITLE
mandatory channels in the Disconnected Setup chapter

### PR DESCRIPTION
**Pull request by keichwa**
# Description
https://bugzilla.suse.com/show_bug.cgi?id=1202464
https://github.com/SUSE/spacewalk/issues/18671
https://github.com/uyuni-project/uyuni-docs/issues/1723

# Target branches

Which documentation version does this PR apply to?

- [x] Master (Default)
- [ ] Manager-4.3
- [ ] Manager-4.2

# Links

Fixes #<insert issue or PR link, if any>

### CODE REVIEW

File: modules/administration/pages/disconnected-setup.adoc [hunk 1]
```diff
@@ -122,6 +122,33 @@ You can change SMT user settings in [path]``/etc/smt.conf``.
...
 
 
+== Mandatory Channels
+
+The corresponding {productname} Client Tools Channels are required, for {productname} to be able to synchronize a given channel.
+If these channels are not enabled, {productname} may fail to detect that product.
+
+Run the following command to enable these mandatory channels:
+
+SLES 12::
```
Comment (mcalmer) [inline modules/administration/pages/disconnected-setup.adoc:132]: Not only SLES, but also all other products based on SLES, like SLES for SAP and HPC, etc.

File: modules/administration/pages/disconnected-setup.adoc [hunk 2]
```diff
@@ -122,6 +122,33 @@ You can change SMT user settings in [path]``/etc/smt.conf``.
...
+RMT: [command]``rmt-cli products enable sle-manager-tools/12/x86_64``
++
+SMT: [command]``smt repos -p sle-manager-tools,12,x86_64``
+
+SLES 15::
```
Comment (mcalmer) [inline modules/administration/pages/disconnected-setup.adoc:137]: same as for SLES12.

File: modules/administration/pages/disconnected-setup.adoc (hunk 1 above)
Comment (keichwa) [reply modules/administration/pages/disconnected-setup.adoc]: ok, thanks!

File: modules/administration/pages/disconnected-setup.adoc (hunk 2 above)
Comment (keichwa) [reply modules/administration/pages/disconnected-setup.adoc]: ok

File: modules/administration/pages/disconnected-setup.adoc [hunk 3]
```diff
@@ -122,6 +122,33 @@ You can change SMT user settings in [path]``/etc/smt.conf``.
...
+If these channels are not enabled, {productname} may fail to detect that product.
+
+Run the following command to enable these mandatory channels:
+
+SLES 12 and products based on it such as SLES for SAP or SLE HPC::
+RMT: [command]``rmt-cli products enable sle-manager-tools/12/x86_64``
++
+SMT: [command]``smt repos -p sle-manager-tools,12,x86_64``
+
+SLES 15 and products based on it such as SLES for SAP or SLE HPC::c
```
Comment (mcalmer) [inline modules/administration/pages/disconnected-setup.adoc:137]: Is the "c" after the `::` is a typo or a special format tag?

**Comment by slsnow** [comment]
That looks good, thanks. 

I did notice there is a bit of an odd space in section 4.3, after RMT: (right before 8.4)

image

File: modules/administration/pages/disconnected-setup.adoc (hunk 3 above)
Comment (keichwa) [reply modules/administration/pages/disconnected-setup.adoc:137]: no, it is a typo.  thanks for spotting!

**Comment by keichwa** [comment]
I found a better solution :)

**Pull request merged**